pkg/services/ngalert/state/manager.go

```diff
@@ -67,6 +69,12 @@ func (st *Manager) Close() {
 }
 
 func (st *Manager) Warm(ctx context.Context) {
+	if st.instanceStore == nil {
```

Comment:
Isn't this quite spammy? From the description, it seems like you're planning to spawn one copy of this for each backtesting call.

Reply:
The Warm method will not be run. I added logs for visibility, if we somehow manage to provide nil reference to any of store in the regular flow.